src/platform/forms/tests/forms.unit.spec.js

```diff
@@ -1,236 +1,303 @@
-            ).to.be.true;
-          });
-        }
-      });
-
-      it('should have chapters object', () => {
-        expect(form.chapters).to.be.an('object');
-      });
-
-      it('should have defaultDefinitions object', () => {
-        expect(form.defaultDefinitions).to.be.an('object');
-      });
-
-      if (form.introduction) {
-        it('should have introduction function', () => {
-          expect(form.introduction).to.be.a('function');
-        });
-      }
-
-      if (form.prefillEnabled) {
-        it('should have prefillEnabled boolean', () => {
-          expect(form.prefillEnabled).to.be.a('boolean');
-        });
-      }
-
-      if (form.prefillTransformer) {
-        it('should have prefillTransformer function', () => {
-          expect(form.prefillTransformer).to.be.a('function');
-        });
-      }
-
-      it('should have trackingPrefix', () => {
-        expect(form.trackingPrefix).to.be.a('string');
-      });
-
-      it('should have title', () => {
-        const title =
-          typeof form.title === 'function'
-            ? form.title({ formData: {} })
-            : form.title;
-        expect(title).to.be.a('string');
-      });
-
-      if (form.subTitle) {
-        it('should have subTitle', () => {
-          expect(form.subTitle).to.be.a('string');
-        });
-      }
-
-      it('should have urlPrefix', () => {
-        expect(form.urlPrefix).to.be.a('string');
-      });
-
-      if (form.submitUrl) {
-        it('should have submitUrl', () => {
-          expect(form.submitUrl).to.be.a('string');
-        });
-      }
-
-      if (form.submit) {
-        it('should have submit', () => {
-          expect(form.submit).to.be.a('function');
-        });
-      }
-
-      if (form.savedFormMessages) {
-        it('should have savedFormMessages', () => {
-          expect(form.savedFormMessages).to.be.a('object');
-        });
-      }
-
-      if (form.transformForSubmit) {
-        it('should have transformForSubmit', () => {
-          expect(form.transformForSubmit).to.be.a('function');
-        });
-      }
-
-      it('should have confirmation', () => {
-        expect(form.confirmation).to.be.a('function');
-      });
-
-      if (form.preSubmitInfo) {
-        it('should have preSubmitInfo', () => {
-          expect(form.preSubmitInfo).to.be.a('object');
-        });
-      }
-
-      if (form.footerContent) {
-        it('should have footerContent', () => {
-          expect(form.footerContent).to.be.a('function');
-        });
-      }
-
-      if (form.getHelp) {
-        it('should have getHelp', () => {
-          expect(form.getHelp).to.be.a('function');
-        });
-      }
-
-      if (form.errorText) {
-        it('should have errorText', () => {
-          expect(form.errorText).to.be.a('function');
-        });
-      }
+  Object.values(configFiles).forEach(configFilePath => {
+    it(`${configFilePath.replace(root, '')}:`, () => {
+      return expect(
```

Comment:
this is needed in order for failing expects within the promise to actually trigger a failure of the test

Reply:
Good call out. Might warrant a comment in the code for posterity.

Reply:
Added comment for this and the `return true` below